components/user/UserTestimonials.js

```diff
@@ -51,4 +44,4 @@ export default function UserTestimonials({ users, data, BASE_URL }) {
         ))}
     </>
   );
-}
+}
```

Comment:
Ending blank line is needed at the end of the file (indicate by GitHub by red icon)

Reply:
Yeah! sure. Done.